packages/server/src/automations/steps/queryRows.ts

```diff
@@ -117,7 +117,13 @@ function typeCoercion(filters: SearchFilters, table: Table) {
           continue
         }
         if (column.type === FieldTypes.NUMBER) {
-          searchParam[property] = parseFloat(value)
+          if (key === "oneOf") {
+            value = value.split(",").map(item => parseFloat(item))
+          } else {
+            value = parseFloat(value)
+          }
+          // @ts-ignore
```

Comment:
why is the `@ts-ignore` needed? generally we prefer any typing to removing all type security (if no typing is possible at all). 

Can probably remove all other `@ts-ignore` from this file - the types should be fully solvable now.

Reply:
ts-ignore removed!